src/styles/theme.tsx

```diff
@@ -7,6 +7,13 @@ export const theme = createTheme({
       lineHeight: '34px',
       fontSize: '32px',
       fontWeight: 700,
+      textAlign: 'center',
+      margin: '10px',
```

Comment:
I think margin and textAlign should not be part of the theme as it's a variable value: https://mui.com/material-ui/customization/default-theme/